plugins/css-prefers-color-scheme/README.md

```diff
@@ -1,95 +1,321 @@
 
-[Prefers Color Scheme] lets you use light and dark color schemes in all
-browsers, following the [Media Queries] specification.
+[Prefers Color Scheme] lets you use light and dark color schemes in all browsers, following the [Media Queries] specification.
 
-[!['Can I use' table](https://caniuse.bitsofco.de/image/prefers-color-scheme.png)](https://caniuse.com/#feat=prefers-color-scheme)
+```pcss
+@media (prefers-color-scheme: dark) {
+	:root {
+		--site-bgcolor: #1b1b1b;
+		--site-color: #fff;
+	}
+}
+
+@media (prefers-color-scheme: light) {
+	:root {
+		--site-bgcolor: #fff;
+		--site-color: #222;
+	}
+}
+
+/* becomes */
+
+@media (color: 48842621) {
+	:root {
+		--site-bgcolor: #1b1b1b;
+		--site-color: #fff;
+	}
+}
+
+@media (prefers-color-scheme: dark) {
+	:root {
+		--site-bgcolor: #1b1b1b;
+		--site-color: #fff;
+	}
+}
+
+@media (color: 70318723) {
+	:root {
+		--site-bgcolor: #fff;
+		--site-color: #222;
+	}
+}
+
+@media (prefers-color-scheme: light) {
+	:root {
+		--site-bgcolor: #fff;
+		--site-color: #222;
+	}
+}
+```
 
 ## Usage
 
-From the command line, transform CSS files that use `prefers-color-scheme`
-media queries:
+Add [Prefers Color Scheme] to your project:
 
 ```bash
-npx css-prefers-color-scheme SOURCE.css TRANSFORMED.css
+npm install postcss css-prefers-color-scheme --save-dev
+```
+
+Use it as a [PostCSS] plugin:
+
+```js
+const postcss = require('postcss');
+const prefersColorScheme = require('css-prefers-color-scheme');
+
+postcss([
+	prefersColorScheme(/* pluginOptions */)
+]).process(YOUR_CSS /*, processOptions */);
 ```
 
-Next, use that transformed CSS with this script:
+[Prefers Color Scheme] runs in all Node environments, with special
+instructions for:
+
+| [Node](INSTALL.md#node) | [PostCSS CLI](INSTALL.md#postcss-cli) | [Webpack](INSTALL.md#webpack) | [Create React App](INSTALL.md#create-react-app) | [Gulp](INSTALL.md#gulp) | [Grunt](INSTALL.md#grunt) |
+| --- | --- | --- | --- | --- | --- |
+
+## Options
+
+### preserve
+
+The `preserve` option determines whether the original notation
+is preserved. By default, it is preserved.
+
+```js
+prefersColorScheme({ preserve: false })
+```
+
+```pcss
+@media (prefers-color-scheme: dark) {
+	:root {
+		--site-bgcolor: #1b1b1b;
+		--site-color: #fff;
+	}
+}
+
+@media (prefers-color-scheme: light) {
+	:root {
+		--site-bgcolor: #fff;
+		--site-color: #222;
+	}
+}
+
+/* becomes */
+
+@media (color: 48842621) {
+	:root {
+		--site-bgcolor: #1b1b1b;
+		--site-color: #fff;
+	}
+}
+
+@media (color: 70318723) {
+	:root {
+		--site-bgcolor: #fff;
+		--site-color: #222;
+	}
+}
+```
+
+## Browser
+
+```js
+// initialize prefersColorScheme (applies the current OS color scheme, if available)
+const prefersColorScheme = require('css-prefers-color-scheme/browser')();
```

Comment:
Is the immediate invocation correct? Also, should we start advocating for `import`?

Reply:
> Is the immediate invocation correct?

It is correct, but I don't think it is very clear. (old docs)

I've updated all these examples to use `import`.
The immediate invocation is now also gone.